File: Android/M-Branch - Lopha/M_branch/src/main/java/com/trimline/m_branch/members/member.java
package com.trimline.m_branch.members;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.Ignore;
import androidx.room.PrimaryKey;

import com.trimline.m_branch.vehicles.vehicles;

/**
 * Created by Paul on 11-Dec-16.
 */
@Entity
public class member {
    @PrimaryKey
    @NonNull
    public String No;
    public String Name;
    public String Phone_No;
    public String ID_No;
    public  int Gender;
    public String E_Mail;
    public String Group;
    public Boolean updated = false;
    public double Un_allocated_Funds;
    public double Loan_Balances;
    public double Repayment;
    public double Outstanding_Penalty;
    public double Operation_Cost;
    @Ignore
    public  com.trimline.m_branch.vehicles.vehicles[] vehicles ;

    public double Loan_Arrears;
    public double dailyrepayment;
    public double Savings;
    public double Xmas;
    public String Last_update_savings;
    public String Last_update_xmas;
    public String Last_update_Loan;
    public String Key;
    public double Deposit ;
    /// <remarks/>
    public double Welfare;
}

File: Android/Investor/app/src/main/java/com/trimline/investor/member.kt
package com.trimline.investor

/**
 * Created by Paul on 11-Dec-16.
 */
abstract class member {
    var No: String? = null
    var Name: String? = null
    var Phone_No: String? = null
    var ID_No: String? = null
    var Gender = 0
    var E_Mail: String? = null
    var Group: String? = null
    var updated = false
    var Un_allocated_Funds = 0.0
    var Loan_Balances = 0.0
    var Repayment = 0.0
    var Outstanding_Penalty = 0.0
    var Operation_Cost = 0.0
    abstract var vehicles: Array<vehicles>
    var Loan_Arrears = 0.0
    var dailyrepayment = 0.0
    var Savings = 0.0
    var Xmas = 0.0
    var Last_update_savings: String? = null
    var Last_update_xmas: String? = null
    var Last_update_Loan: String? = null
    var Key: String? = null
}